exercise/templates/exercise/staff/_submissions_table_compact.html

```diff
@@ -56,11 +56,19 @@
 				</span>
 
 				{% if s == summary.best_submission %}
-				<span
-					class="label label-primary"
-					data-toggle="tooltip"
-					title="{% translate 'SUBMISSION_FINAL_TOOLTIP' %}"
-				>{% translate 'FINAL' %}</span>
+					{% if s.force_exercise_points %}
+					<span
+						class="label label-primary"
+						data-toggle="tooltip"
+						title="{% translate 'SUBMISSION_FINAL_TOOLTIP' %}"
+					>{% translate 'FINAL' %}</span>
+					{% else %}
+					<span
+						class="glyphicon glyphicon-star"
+						data-toggle="tooltip"
+						title="{% translate 'SUBMISSION_CHOSEN_TOOLTIP' %}"
+					></span>
```

Comment:
This isn't a huge issue, but I think the gray star next to a colorful label looks a bit awkward. Could it perhaps be part of the points label or have its own colored label next to it?

Reply:
Now including the star inside a blue label. Any better?

Reply:
To be honest I think this looks way worse :joy:
The star is off center and the label's height doesn't match the points label next to it. Plus it looks like the Somalian flag. I think we should go with your earlier version, it looked good enough.